City-of-Helsinki/drupal-helfi-platform-config
# UHF-885 Cookie page

How to test:
* If you don't already have a site for testing, create one:
  * `composer create-project City-of-Helsinki/drupal-helfi-platform:dev-main hel-platform --no-interaction --repository https://repository.drupal.hel.ninja/`
  * `cd hel-platform`
* Checkout correct branches: `composer require drupal/helfi_platform_config:dev-UHF-885_cookie_page && composer require drupal/hdbt:dev-UHF-885_cookie_page`
* `make new`

or in kymp/sote instance: 
* `make new` or `make fresh` ( You can ask for a dump.sql file from @khalima )
  * Checkout `UHF-885_cookie_page` branch
  * Run `composer require drupal/helfi_platform_config:dev-UHF-885_cookie_page && composer require drupal/hdbt:dev-UHF-885_cookie_page`
  * Run `make drush-deploy`

* Go to: https://hel-platform.docker.so/fi/cookie-information-and-settings
  * Check that the page looks prettier than [this page](https://nginx-helfi-dev.agw.arodevtest.hel.fi/fi/cookie-information-and-settings)
  * Check that the page looks prettier in mobile than [this page](https://nginx-helfi-dev.agw.arodevtest.hel.fi/fi/cookie-information-and-settings)
  * Check that Finnish, Swedish and English translations exist
 
Also check these:
https://github.com/City-of-Helsinki/drupal-hdbt/pull/152
https://github.com/City-of-Helsinki/drupal-helfi/pull/195
https://github.com/City-of-Helsinki/drupal-helfi-kymp/pull/88
https://github.com/City-of-Helsinki/drupal-helfi-sote/pull/94
https://github.com/City-of-Helsinki/drupal-helfi-strategia/pull/42
